### DISCUSSION TITLE
Error on Rust files

### DISCUSSION THREAD

**Discussion by augustocdias** [nvim-treesitter/nvim-treesitter#5769] [Q&A] [answered]
I'm using neovim nightly and whenever I open a rust file I'm seeing this error:

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/1080789/c499c256-a202-4458-a21b-445f7a899cb7)

How do I check if I'm doing something wrong or if its a bug? It keeps spawning this error so often that I can't even use the editor once it starts.

**Comment by clason** [comment, marked as answer]
https://github.com/nvim-treesitter/nvim-treesitter/issues/3092

**Reply by augustocdias** [reply]
Thanks. That was it :)

**Reply by clason** [reply]
That's why it's a pinned issue ;)
